cartoframes/data/services/bq_datasets.py

```diff
@@ -127,6 +127,29 @@ def create(self, payload):
 
         return response
 
+    def enrichment(self, payload):
+        url = '{}/enrichment'.format(DO_ENRICHMENT_API_URL)
```

Comment:
I changed the URL to `/bq/enrichment`

Reply:
`DO_ENRICHMENT_API_URL = 'http://localhost:7070/bq'`

Reply:
:zipper_mouth_face:

Reply:
That API URL should be:
- `https://{username}.carto.com/api/v4/do/dev/bq` for prod
- `https://{username}.carto-staging.com/api/v4/do/dev/bq` for staging
- `http://{username}.localhost.lan:7070/bq` (or something similar) for development

Reply:
It should be changed first in `data-observatory`, shouldn't it?